examples/goldilocks_models/find_goldilocks_models.cc

```diff
@@ -116,6 +121,8 @@ DEFINE_double(fail_threshold, 0.2,
 DEFINE_bool(get_good_sol_from_adjacent_sample, true,
             "Get a good solution from adjacent samples to improve the solution "
             "quality of the current sample");
+DEFINE_bool(use_theta_gamma_from_files,false,
```

Comment:
This looks to me like a flag for testing only? If so, we should remove it.

Reply:
removed.